### DISCUSSION TITLE
Apply to Github education

### DISCUSSION THREAD

**Discussion by Ukrainets-Oleksii** [community/community#144877] [GitHub Education]
### Hi everyone,

Hey, I have a question. I'm a student and I want to apply to GitHub Education. GitHub Educations list has my Uni but next step I should upload my documents from Uni it's not problem, but I should take a picture, I can't do it because I'm remote studying and I have only PDF documents, how should I take a picture if I have a PDF. Maybe can I upload PDF or talk with support?

**Comment by FeiSuoerDe** [comment]
As far as I know, under the current circumstances, perhaps your PDF document should be printed. Currently, file uploading is not supported.

**Reply by Ukrainets-Oleksii** [reply]
Okay, I'll try thx)

**Reply by FeiSuoerDe** [reply]
> 好的，我会试试 thx）

Perhaps the official is considering safety aspects, but only being able to take pictures is really annoying.
The me who is fighting with the inferior front camera is like：😡
